26-map-filter-reduce/__test__/mfr.test.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+const higherFunc = require('../solution.js');
+require('jest');
+
+let arr = [1,2,3];
+let arrTwo = [1,2,3,4,5,6,7,8,9];
+let func = function(el) {
```

Comment:
make these function names more specific so it is easier to read. For example, call this one `addTwo()` or something similar.